packages/std/src/deps.rs

```diff
@@ -22,3 +23,43 @@ impl<S: Storage, A: Api, Q: Querier> Deps<S, A, Q> {
         }
     }
 }
+
+pub struct DepsMut<'a> {
+    pub storage: &'a mut dyn Storage,
+    pub api: &'a dyn Api,
+    pub querier: QuerierWrapper<'a>,
+}
+
+pub struct DepsRef<'a> {
+    pub storage: &'a dyn Storage,
+    pub api: &'a dyn Api,
+    pub querier: QuerierWrapper<'a>,
+}
```

Comment:
Both of those can implement `Copy`. This probably makes contract development easier when they are used in branched code.

Reply:
I can do that. I was going to add more Copy, then realized the `&mut` cannot be. But sure, the `DepsRef` version can have that, as can the QuerierWrapper

Reply:
Good point, yeah. Let's only do the easy ones.